File: Obnovljivac.API/ClientApp/src/app/models/autogenerated/windDailyEnergyDto.ts

// ----------------------------------------------
//  Interface and model
// ----------------------------------------------
export interface IWindDailyEnergyDto  {
    dailyEnergy: number[][];
    monthlyEnergy: number[];
    yearlyEnergy: number;
}

export class WindDailyEnergyDto  implements IWindDailyEnergyDto {
    public dailyEnergy: number[][];
    public monthlyEnergy: number[];
    public yearlyEnergy: number;

    constructor(dailyEnergy: number[][] = null,  monthlyEnergy: number[] = null,  yearlyEnergy: number = null ) {

        this.dailyEnergy = dailyEnergy;
        this.monthlyEnergy = monthlyEnergy;
        this.yearlyEnergy = yearlyEnergy;
    }
}

File: Obnovljivac.API/Models/DTOs/WindDailyEnergyDto.cs
namespace Obnovljivac.API.Models.DTOs
{
    public class WindDailyEnergyDto
    {
        //public Dictionary<int, Dictionary<int, double>> DailyEnergy { get; set; }
        public List<List<double>> DailyEnergy { get; set; }
        public List<double> MonthlyEnergy { get; set; }
        public double YearlyEnergy { get; set; }
    }
}
